pkgs/tools/graphics/vulkan-tools-lunarg/default.nix

```diff
@@ -0,0 +1,88 @@
+{ stdenv, cmake, expat, fetchFromGitHub, jq, lib, libXdmcp, libXrandr, libffi
+, libxcb, pkgconfig, python3, symlinkJoin, vulkan-headers, vulkan-loader
+, vulkan-validation-layers, wayland, writeText, xcbutilkeysyms, xcbutilwm
+, xlibsWrapper }:
+
+let version = "1.2.141.0";
+
+in
+
+assert version == vulkan-headers.version;
```

Comment:
Usually it's better to put assertions near where the variables are evaluated, so an update to vulikan-headers won't break this packages before it will all be evaluated. So I'd propose to remove this line and instead, (see a followup comment).